### PR TITLE
VIH-8072 Ejudiciary is defined on user name not on contact email

### PR DESCRIPTION
**Pull request by PrasannaDommalapati**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-8072

### Change description ###

- whether a participant is an ejud or not is defined by user name not by contact email

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by robertbennett1998 — DISMISSED** [review]
LGTM

File: AdminWebsite/AdminWebsite.UnitTests/Mappers/NotificationMappers/MapToDemoOrTestNotificationTests.cs [hunk 1]
```diff
@@ -187,7 +187,7 @@ public void Should_map_ejud_joh_demo_or_test_notification()
             var participant = new ParticipantResponse
             {
                 Id = Guid.NewGuid(),
-                Username = "testusername@hmcts.net",
+                Username = "contact@judiciary.hmcts.net",
                 CaseRoleName = "caserolename",
                 ContactEmail = "contact@judiciary.hmcts.net",
```
Comment (kevin-hmcts) [inline AdminWebsite/AdminWebsite.UnitTests/Mappers/NotificationMappers/MapToDemoOrTestNotificationTests.cs:192]: Consider using a different value for ContactEmail to Username (Multiple instances), ideally we want each value to be unique unless there is a need for there to be duplicates.

File: AdminWebsite/AdminWebsite.UnitTests/Extensions/HearingDetailsResponseExtensionsTests.cs [hunk 2]
```diff
@@ -69,79 +69,79 @@ public void Should_Return_False_If_Judge_Email_is_null()
...
                 Id = Guid.NewGuid(),
-                Participants = new List<ParticipantResponse> {newEJudJudge}
+                Participants = new List<ParticipantResponse> { newEJudJudge }
             };
             _hearing.HasJudgeEmailChanged(newHearing).Should().BeTrue();
         }
-        
+
         [Test]
         public void should_return_true_when_ejud_email_has_changed_to_vh_judge()
```
Comment (kevin-hmcts) [inline AdminWebsite/AdminWebsite.UnitTests/Extensions/HearingDetailsResponseExtensionsTests.cs:142]: Consider using Pascal Case, for test names, eg:
ShouldReturnTrue_WhenEjudEmailHasChangedToVHJudge helps to improve readability or 
MethodUnderTest_ShouldReturnTrue_WhenEjudEmailHasChangedToVHJudge could also be added for additional context when test is viewed in isolation.

**Review by kevin-hmcts — APPROVED** [review]
LGTM

**Review by robertbennett1998 — APPROVED** [review]
LGTM

File: AdminWebsite/AdminWebsite.UnitTests/Extensions/HearingDetailsResponseExtensionsTests.cs (hunk 2 above)
Comment (PrasannaDommalapati) [reply AdminWebsite/AdminWebsite.UnitTests/Extensions/HearingDetailsResponseExtensionsTests.cs:142]: these tests were inherited, we will write in the pascal case.

File: AdminWebsite/AdminWebsite.UnitTests/Mappers/NotificationMappers/MapToDemoOrTestNotificationTests.cs (hunk 1 above)
Comment (PrasannaDommalapati) [reply AdminWebsite/AdminWebsite.UnitTests/Mappers/NotificationMappers/MapToDemoOrTestNotificationTests.cs:192]: Sure.

**Pull request merged**